opm/autodiff/BlackoilModelParametersEbos.hpp

```diff
@@ -72,7 +72,7 @@ SET_BOOL_PROP(FlowModelParameters, UpdateEquationsScaling, false);
 SET_BOOL_PROP(FlowModelParameters, UseUpdateStabilization, true);
 SET_BOOL_PROP(FlowModelParameters, MatrixAddWellContributions, false);
 SET_SCALAR_PROP(FlowModelParameters, TolerancePressureMsWells, 0.01 *1e5);
-SET_SCALAR_PROP(FlowModelParameters, MaxPressureChangeMsWells, 2.0 *1e5);
+SET_SCALAR_PROP(FlowModelParameters, MaxPressureChangeMsWells, 1e6);
 SET_BOOL_PROP(FlowModelParameters, UseInnerIterationsMsWells, true);
```

Comment:
@GitPaean This could affect performance, even thought I havn't notice any regression.

Reply:
I will test it separately. Hopefully it only does good.